manifests/crds/argoproj.io_applicationsets.yaml

```diff
@@ -475,6 +475,15 @@ spec:
                           - path
                           type: object
                         type: array
+                      files:
```

Comment:
Just curious ... does this change related to the goal of the PR?

Reply:
Since we've now added the `check-manifests` check above, this caused this file to fail due to being out-of-sync with the output of `make manifests`.

This file is likely out of sync due to a previous PR, and here I needed to clean it up in order to make `check-manifests` happy. Fortunately with this check in place, it should prevent that in the future.

Reply:
Looks like this same mismatch was also fixed in https://github.com/argoproj-labs/applicationset/pull/91, which I just merged, so this block is now no longer needed.